app/models/user.rb

```diff
@@ -1,14 +1,17 @@
-# frozen_string_literal: true
-
 class User < ActiveRecord::Base
 
-  # lo tuve que agregar a mano
-  #extend Devise::Models
 
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
-        :recoverable, :rememberable, :trackable, :validatable
+        :recoverable, :rememberable, :validatable
   include DeviseTokenAuth::Concerns::User
+
+  validates :targets, length: {
```

Comment:
This validation looks nice but the issue is that as this is a validation in the user, this won't be run when new targets are created so you need to do this validation in target with a validation defined by you